examples/user_interfaces/embedding_in_wx5.py

```diff
@@ -38,15 +48,15 @@ def add(self, name="plot"):
 
 
 def demo():
-    app = wx.PySimpleApp()
+    # alternatively you could use
+    #app = wx.App()
```

Comment:
This comment might need more explanation. As it stands, the user is left wondering what the difference is.